amgcl/backend/vexcl_static_matrix.hpp

```diff
@@ -778,12 +778,18 @@ struct vex_mul {
     } const apply;
 };
 
-template <typename Alpha, typename Beta, typename TA, typename TX, typename TY, int B>
+template <typename Alpha, typename Beta, typename TA, typename TX, typename TY, int B, typename C, typename P>
 struct spmv_impl<Alpha,
-    vex::sparse::distributed<vex::sparse::matrix<static_matrix<TA,B,B>, ptrdiff_t, ptrdiff_t>>,
-    vex::vector<static_matrix<TX,B,1>>, Beta, vex::vector<static_matrix<TY,B,1>>>
+    vex::sparse::distributed<vex::sparse::matrix<static_matrix<TA,B,B>, C, P>>,
+    vex::vector<static_matrix<TX,B,1>>, Beta, vex::vector<static_matrix<TY,B,1>>,
+    typename std::enable_if<
+        (math::static_rows<static_matrix<TA,B,B>>::value > 1) &&
```

Comment:
would single `B>1` condition be enough here?

Reply:
Yes, sure!